raiden/api/rest.py

```diff
@@ -258,12 +259,13 @@ def open(self, partner_address, token_address, settle_timeout, balance=None):
                 )
             except EthNodeCommunicationError as e:
                 return make_response(str(e), httplib.REQUEST_TIMEOUT)
+            except InsufficientFunds as e:
+                return make_response(str(e), httplib.CONFLICT)
```

Comment:
According to some quick googling (e.g. https://news.ycombinator.com/item?id=7857236) it's being used despite the "reserved" state.